readme/BREAKING_CHANGES.markdown

```diff
@@ -59,6 +59,30 @@ horizontal rule):
 ---------------------------------------
 
 ```
+
+### Remove CSS Compatibility Layer
+- **Date:** January 2021
```

Comment:
Hey @marcoscv-work, you add new entries at the end of the document, not at the beginning 😉